wpilibc/src/main/native/cpp/shuffleboard/Shuffleboard.cpp

```diff
@@ -63,10 +63,25 @@ void Shuffleboard::AddEventMarker(std::string_view name,
   AddEventMarker(name, "", importance);
 }
 
+static std::unique_ptr<detail::ShuffleboardInstance>& GetInstanceHolder() {
+  static std::unique_ptr<detail::ShuffleboardInstance> instance =
+      std::make_unique<detail::ShuffleboardInstance>(
+          nt::NetworkTableInstance::GetDefault());
+  return instance;
+}
+
+#ifndef __FRC_ROBORIO__
+namespace frc::impl {
+void ResetShuffleboardInstance() {
+  std::make_unique<detail::ShuffleboardInstance>(
+      nt::NetworkTableInstance::GetDefault())
+      .swap(GetInstanceHolder());
```

Comment:
Since it returns a mutable reference, you should be able to simply assign it rather than doing a swap.